Sources/SourceKitLSPAPI/BuildDescription.swift

```diff
@@ -23,13 +23,16 @@ import class Build.ClangTargetBuildDescription
 import class Build.SwiftTargetBuildDescription
 import struct PackageGraph.ResolvedModule
 import struct PackageGraph.ModulesGraph
+import enum PackageGraph.BuildTriple
```

Comment:
@MaxDesiatov Do you prefer to expose the `BuildTriple` type or would you prefer to define an equivalent `BuildTriple` type in `SourceKitLSPAPI` and then map between the two?

Reply:
For now IMO it's best to create `public typealias BuildTriple = PackageGraph.BuildTriple` in `SourceKitLSPAPI`, which can be replaced with something more advanced later.